packages/node_modules/@webex/widget-call-history/src/utils/dateUtils.ts

```diff
@@ -127,3 +127,37 @@ export function formatDateDDMMYYYY(date: string) {
     year: 'numeric',
   });
 }
+
+/**
+ * @description Handle formatting the date for announce.
+ * @param {string} date The date string to format
+ * @returns {string} A announced date as in "Today", "Yesterday", or "june 10,2024"
+ */
+export const formatDateForAnnouncement = (dateValue: string) => {
+  const date = new Date(dateValue);
+  const today = new Date();
+  const yesterday = new Date();
+  yesterday.setDate(yesterday.getDate() - 1);
+
+  if (isSameDate(date, today)) {
+      return "Today";
+  }
+  if (isSameDate(date, yesterday)) {
+      return "Yesterday";
```

Comment:
Same question here also.
How do we manage for other languages?

Reply:
will be updating the code to supports other language